### ISSUE TITLE
KODI/SMPC for Windows OS?

### ISSUE THREAD

**Issue by NelsonJfsg** [koying/SPMC#994]
I wanted to know if there is a version for windows?

**Comment by nexusV2** [comment]
Yes here it is [http://download.semperpax.com/spmc/](http://download.semperpax.com/spmc/)

**Comment by unlocKing** [comment]
https://github.com/koying/SPMC/issues/984
